packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.txt

```diff
@@ -0,0 +1,8 @@
+android-download-plaintext = "Get <%- locals.productName %> on Google Play:"
+<%- locals.androidUrl %>
+
+ios-download-plaintext = "Get <%- locals.productName  %> on the App Store:"
+<%- locals.iosUrl %>
+
+another-device-plaintext = "Get <%- locals.productName %> on another device:"
```

Comment:
Per the [doc](https://docs.google.com/document/d/18gF_7Ux-jE_qfEFciOcPml0-G1SfHkjiJSMPwh2hnSc/edit#heading=h.q439jqhxcydz) This should say 'Install Firefox' not 'Get Firefox'